webproject/src/main/java/com/google/sps/data/Order.java

```diff
@@ -0,0 +1,60 @@
+package com.google.sps.data;
+
+public class Order {
+    private String user;
```

Comment:
I think this should be userId, or email, if you want to use the email as unique identifier

Reply:
Yup renamed it and used userService for all